rules-tests/TypeDeclaration/Rector/ClassMethod/ReturnTypeFromStrictParamRector/Fixture/skip_parent_overridden.php.inc

```diff
@@ -1,15 +0,0 @@
-<?php
-
-namespace Rector\Tests\TypeDeclaration\Rector\ClassMethod\ReturnTypeFromStrictParamRector\Fixture;
-
-class SkipParentOverridden extends SkipParentOverriddenBase {
-    public function doFoo(SkipParentOverridden $param) {
-        return $param;
-    }
-}
-
-
-class SkipParentOverriddenBase {
-    public function doFoo(SkipParentOverridden $param) {
```

Comment:
I feel this test-fixture did miss a `SkipParentOverridden` incompatible return type at this point, e.g. `int`

Reply:
it is about override return, the param is equal :)

Reply:
what I try to say: we shouldn't delete this fixture, but keep it and just add a `:int` return type in the base method class

Reply:
ok, I added parent already has return fixture https://github.com/rectorphp/rector-src/pull/4878/commits/397fc3aecd996b2bfbfc13399a50ce84605205e0